src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -228,6 +228,17 @@ export const ShippingRoute: FC<ShippingProps> = props => {
       : addressVerificationIntlEnabled
   }
 
+  // Save shipping info on the order. If it's Artsy shipping and a quote hasn't
+  // been selected, this renders the quotes for user to select and finalize
+  // again.
```

Comment:
This step is actually pretty complicated (and so hard to name it) and can use some refactor. I'm documenting it first and would like to do another followup.